src/transformers/utils/hub.py

```diff
@@ -1081,6 +1093,27 @@ def extract_info_from_url(url):
     return {"repo": cache_repo, "revision": revision, "filename": filename}
 
 
+def create_and_tag_model_card(repo_id, tags=None):
+    """
+    Creates a dummy model card and tags it.
+    """
+    model_card = None
+
+    try:
+        # Check if the model card is present on the remote repo
+        model_card = ModelCard.load(repo_id)
+    except EntryNotFoundError:
+        # Otherwise create a simple model card from template
+        card_data = ModelCardData(language="en", tags=[])
```

Comment:
(not sure) is it expected that all models should be tagged as an "English" model?

Also could we add `library_name` and `model_name` as well?

Reply:
Done ! I also removed the default `en` language tag